pimd/pim_msdp.h

```diff
@@ -222,12 +222,6 @@ struct pim_msdp {
 struct pim_instance;
 void pim_msdp_init(struct pim_instance *pim, struct thread_master *master);
 void pim_msdp_exit(struct pim_instance *pim);
-enum pim_msdp_err pim_msdp_peer_add(struct pim_instance *pim,
```

Comment:
Add/del are the extern functions why not keep and modify them in place? Also new/do_del seem like static/internal function names so if a wrapper is not needed on top of them maybe collapse them into peer_add and peer_del and use them for that purpose adding/deleting peers (i.e. just re-name peer_new=>peer_add and peer_do_del=>del).

Reply:
Fixed, see last commit.